samples/web/content/apprtc/index.html

```diff
@@ -58,6 +58,11 @@
     </div>
   </div>
 
+  <div id="confirm-join-div" class="hidden">
+    <div> Are you ready to join <span id="confirm-join-room-span"></span>?</div>
```

Comment:
If the room is all numeric, or perhaps 8 chars and all numeric, I think that would be fine for now.

The long-term solution here is to generate rooms with random words, not random numbers.